utilities/Constants.ts

```diff
@@ -22,7 +22,9 @@ export const MFA_OPTIONS = [
   }
 ];
 
-export const COJ_EPOCH = new Date("2023-08-01");
+export const COJ_EPOCH = new Date(
+  process.env.NEXT_PUBLIC_CONDITIONS_OF_JOINING_EPOCH as string
+);
```

Comment:
It feels a bit awkward having a "constant" set via env vars, but we have a real issue when it comes to testing both locally and on stage. This allows setting the "COJ epoch" per environment and allows us to verify the stage deployment without any risk of enabling COJ on prod.